### PR TITLE
🐺 Add --qawolf flag

### LINKED ISSUE #1: Don't assume tests
Maybe have npm test commented out and include node myScript.js commented out too

### PR DESCRIPTION
**Pull request by flaurida**
Resolves #1

### CODE REVIEW

File: static/azure.hbs [hunk 1]
```diff
@@ -21,4 +21,10 @@ steps:
...
   # - script: npm run start & npx wait-on http://localhost:3000
 
-  - script: npm test
\ No newline at end of file
+  {{#if qawolf}}
+  - script: npx qawolf test
+  {{else}}
+  # replace below with command you want to run, example for running a script below
+  # - script: node myScript.js
+  - script: npm test
```
Comment (jperl) [inline static/azure.hbs]: I would just commend these both out

File: static/azure.hbs [hunk 2]
```diff
@@ -21,4 +21,10 @@ steps:
   # # Start local server
   # - script: npm run start & npx wait-on http://localhost:3000
 
-  - script: npm test
\ No newline at end of file
+  {{#if qawolf}}
+  - script: npx qawolf test
+  {{else}}
+  # replace below with command you want to run, example for running a script below
```
Comment (jperl) [inline static/azure.hbs:27]: I would change this to: "example for running a script/test below"

and have the npm test commented out too

Comment (flaurida) [reply static/azure.hbs:27]: I'm in the camp of keeping the default obvious, they can change if they want

File: src/ci.ts [hunk 3]
```diff
@@ -21,8 +21,16 @@ const paths = {
...
 };
 
-export const saveCiTemplate = async (provider: CiProvider): Promise<void> => {
-  const providerPath = paths[provider];
+const qawolfPaths = {
+  ...paths,
+  github: '.github/workflows/qawolf.yml',
+};
+
+export const saveCiTemplate = async (
```
Comment (jperl) [inline src/ci.ts]: just one options object with both 🙏. will be more flexible for the future

**Review by jperl — APPROVED** [review]

File: static/azure.hbs (hunk 2 above)
Comment (jperl) [reply static/azure.hbs:27]: Ok 👍

**Pull request merged**